src/cli_plugin/install/__tests__/download.js

```diff
@@ -0,0 +1,226 @@
+import expect from 'expect.js';
+import sinon from 'sinon';
+import nock from 'nock';
+import glob from 'glob-all';
+import rimraf from 'rimraf';
+import mkdirp from 'mkdirp';
+import Logger from '../../lib/logger';
+import { download, _downloadSingle } from '../download';
+import { join } from 'path';
+
+describe('kibana cli', function () {
+
+  describe('plugin downloader', function () {
+    const testWorkingPath = join(__dirname, '.test.data');
```

Comment:
Not a big deal, but it'd probably be better to dump this sort of stuff into the operating system tmp directory rather than writing to the project itself.  The [tmp](https://www.npmjs.com/package/tmp) module is awesome for this as it'll automatically clean up when the process is killed.